cluster/manifests/aws-cloud-controller-manager/rbac.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cloud-controller-manager
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: aws-cloud-controller-manager
```

Comment:
Don't we need an IAM role for it or how does it get permission to access AWS?

Reply:
Good point!

It uses `hostNetwork` (taken from upstream example) so it's using the role of the node, but it makes sense to use a dedicated role ofc.

Reply:
Changed to use dedicated IAM role and not `hostNetwork`.